### PR TITLE
Add documentation for celltype refs files

### LINKED ISSUE #389: Document references folder
Since `references` now contains references of two different flavors for for two different workflows (e.g. transcriptome references vs cell type annotation references), time to add a quick `references/README.md`. (Later, we'll want more docs about, for example, adding more references to this workflow; can we open an issue for that?) For this `README.md`, here's some particular aspects I think we should make sure are there, beyond the baseline "what are these files" content - 
	- Indicate that the `organ` column in `celltype-reference-metadata.tsv` has relevant values from the `organ` column in the PanglaoDB
	- References (no pun intended, probably, ok maybe a little) for `celldex` and `PanglaoDB`

_Originally posted by @sjspielman in https://github.com/AlexsLemonade/scpca-nf/pull/388#pullrequestreview-1541404117_

### PR DESCRIPTION
**Pull request by allyhawkins**
Closes #389 

So I went to document the `references` folder per #389 and was pleasantly surprised that I had already documented the files related to references for the main `scpca-nf`. So I just had to add in some context about the two new files related to building cell type references. 

I added on to the section that I already had in our internal documentation describing the two files. Generally this was pretty straight forward but let me know if I need to expand on anything!

### CODE REVIEW

File: internal-instructions.md [hunk 1]
```diff
@@ -67,6 +67,17 @@ See [instructions for adding additional organisms](#adding-additional-organisms)
 For each supported reference, a list of all the reference files that are needed to run `scpca-nf` will be included.
 This file is required as input to `scpca-nf`.
 
+3. `celltype-reference-metadata.tsv`: Each row of this TSV file corresponds to a supported cell type reference available for cell type assignment using `add-celltypes.nf`.
+For all references, the following columns will be populated: `celltype_ref_name`, `celltype_ref_source` (e.g., `celldex`), supported `celltype_method` (e.g., `SingleR`).
+For all references obtained from the `PanglaoDB` source also require an `organs` column containing the list of supported `PanglaoDB` organs to include when building the reference.
+This should be a comma separated list of all organs to include.
```
Comment (sjspielman) [inline internal-instructions.md:73]:
```suggestion
This should be a comma-separated list of all organs to include.
```

**Review by sjspielman — COMMENTED** [review]
Changes look good to me! The one additional thing I think would be good to also have is a line of "how to run the workflow", aka a quick code chunk showing how to run `build-celltype-ref.nf`.

**Comment by allyhawkins** [comment]
> Changes look good to me! The one additional thing I think would be good to also have is a line of "how to run the workflow", aka a quick code chunk showing how to run build-celltype-ref.nf.

File: internal-instructions.md [hunk 2]
```diff
@@ -88,3 +100,11 @@ homo_sapiens
 3. Generate an updated `scpca-refs.json` by running the script, `create-reference-json.R`, located in the `scripts` directory.
 4. Generate the index files using `nextflow run build-index.nf -profile ccdl,batch` from the root directory of this repository.
 5. Ensure that the new reference files are public and in the correct location on S3 (`s3://scpca-references`).
+
+## Adding additional cell type references
+
+Follow the below steps to add suppor for additional cell type references:
```
Comment (sjspielman) [inline internal-instructions.md:106]: fixes a typo (suppor**t**), and adds sentence which you can feel free to tweak!
```suggestion
Follow the below steps to add support for additional cell type references.
We currently only support `celldex` and `PanglaoDB` for reference sources.
```

**Review by sjspielman — APPROVED** [review]
👍 
Just one comment fixing a typo, and an additional sentence you can take or leave, up to you!

**Pull request merged**